### DISCUSSION TITLE
add  'Cancel' in rename popup

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#7772] [Configuration]
<img width="602" alt="Schermafbeelding 2020-11-22 om 17 05 33" src="https://user-images.githubusercontent.com/33354141/99908855-52031a00-2ce5-11eb-835f-88d47e0e19ce.png">

Please consider adding a 'Cancel' here (/configuration/devices/device), for the situation one has already set a new name, but decides (maybe too late) it shouldn't happen after all. 
Right now, we can only click No (in which case the device only gets renamed, or yes, in which case all related entities get renamed too.

There's no going back now, which would be really helpful, (as I've just discovered....)

**Comment by KTibow** [comment]
Couldn't you repeat it with the old name?

**Comment by Mariusthvdb** [comment]
havent tried, but I guess that would be possible.
still, that is a very user unfriendly solution, where a cancel button would suffice.
compared to an online purchase and you get my drift ;-)

**Reply by KTibow** [reply]
Well then probably you should add another dialog / change the name of it. Can you think of any ways how changing a device name might break something?
